addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -540,9 +540,10 @@ const Wysiwyg = Widget.extend({
                 const $btn = this.toolbar.$el.find('#create-link');
                 this.linkTools = new weWidgets.LinkTools(this, {wysiwyg: this}, this.odooEditor.editable, {}, $btn, options.link);
                 const _onMousedown = ev => {
-                    if (!ev.target.closest('.oe-toolbar') && !ev.target.closest('.ui-autocomplete')) {
+                    if (!ev.target.closest('.oe-toolbar') && !ev.target.closest('.ui-autocomplete')
+                        && !ev.target.closest('.o_technical_modal')) {
                         // Destroy the link tools on click anywhere outside the
-                        // toolbar.
+                        // toolbar (except on dialog).
```

Comment:
I don't see the effect of this? I click on "upload a file", the dialog opens, I click in the seachbar -> the link tools are disappearing. Is it supposed to solve that?

Reply:
Without this, dialog was closed by any click (file select, button, url entry) that's why I did it. I had the idea to prevent the link tool from closing but there were issues with the selection. I'll try it again !